platform-includes/getting-started-config/javascript.nuxt.mdx

```diff
@@ -69,8 +69,33 @@ dotenv.config();
 // ... rest of the file
 ```
 
+#### Troubleshoot Server-Side
 
 <Alert level="warning">
   In the beta state of the Nuxt SDK, some features may not work with certain deployment providers. Check the progress on GitHub: [Compatibility with different Deployment Platforms](https://github.com/getsentry/sentry-javascript/issues/14029)
 </Alert>
 
+When adding `sentry.server.config.ts`, you might get an error like this:
+`Failed to register ESM hook import-in-the-middle/hook.mjs`. You can add an override (npm/pnpm) or a resolution (yarn)
+for `@vercel/nft` to fix this. This will add the `hook.mjs` file to your build output
+([Nitro issue here](https://github.com/unjs/nitro/issues/2703)).
```

Comment:
```suggestion
for `@vercel/nft` to fix this. This will add the `hook.mjs` file to your build output.
See the [underlying issue in the UnJS Nitro project](https://github.com/unjs/nitro/issues/2703).
```